src/Driver.hpp

```diff
@@ -4,6 +4,7 @@
 #include <iodrivers_base/Driver.hpp>
 #include <marnav/nmea/nmea.hpp>
 #include <marnav/nmea/sentence.hpp>
+#include <nmea0183/Exceptions.hpp>
```

Comment:
Because it defines an exception that is thrown by the class. Without this, every file that includes Driver.hpp would also need to include Exceptions.hpp to handle the exception.